### ISSUE TITLE
layout bug w/ long list names

### ISSUE THREAD

**Issue by debergalis** [percolatestudio/mdg-todos#43]
![screen shot 2014-10-07 at 12 33 28 pm](https://cloud.githubusercontent.com/assets/204768/4548894/e2d22d04-4e58-11e4-9557-ae47f4bdd84e.png)

**Comment by tmeasday** [comment]
Note that to reproduce this you need 
  a) the exact right length text (any longer and it'll wrap),
  b) at least 10 items.

**Comment by tmeasday** [comment]
Pushed a work around that moves the number slightly to the right. @dominicnguyen when you have a chance take a look at this.

Note that the problem will probably occur if you have 100+ items.

**Comment by stubailo** [comment]
Is it possible to do a complete fix by putting the todos count element before the list title? AFAIK that would make the list title wrap around the count.

**Comment by tmeasday** [comment]
You're right @stubailo, that's a much better fix. @dominicnguyen just make sure you are happy with this.

**Comment by domyen** [comment]
@tmeasday The left-hand side is the space that denotes the state of list privacy –when a list is private we show an icon here. I'll fix the bug and revert your change.

**Comment by stubailo** [comment]
By "before" I meant before in the HTML markup, not in the resulting UI. float: right will still make the number show up to the right, I think.

**Comment by domyen** [comment]
Woops, misunderstanding. Reverting the revert & good suggestion :)
